client-app/src/examples/portfolio/PortfolioPanelModel.js

```diff
@@ -1,40 +1,54 @@
 import {HoistModel, LoadSupport, managed, loadAllAsync} from '@xh/hoist/core';
 import {PositionsPanelModel} from './PositionsPanelModel';
-import {OrdersPanelModel} from './OrdersPanelModel';
-import {LineChartModel} from './LineChartModel';
-import {OHLCChartModel} from './OHLCChartModel';
-import {bindable} from '@xh/hoist/mobx';
+import {SplitTreeMapModel} from '@xh/hoist/desktop/cmp/treemap';
+import {hspacer} from '@xh/hoist/cmp/layout';
+import {fmtMillions} from '@xh/hoist/format';
+import {PositionInfoPanelModel} from './PositionInfoPanelModel';
 
 @HoistModel
 @LoadSupport
 export class PortfolioPanelModel {
 
     @managed positionsPanelModel = new PositionsPanelModel();
-    @managed ordersPanelModel = new OrdersPanelModel();
-    @managed lineChartModel = new LineChartModel();
-    @managed ohlcChartModel = new OHLCChartModel();
+    @managed splitTreeMapModel = new SplitTreeMapModel({
+        gridModel: this.positionsPanelModel.gridModel,
+        mapFilter: rec => rec.pnl >= 0,
+        mapTitleFn: (mapName, model) => {
+            const isPrimary = mapName === 'primary',
+                v = isPrimary ? model.primaryMapTotal : model.secondaryMapTotal;
+            return [
+                isPrimary ? 'Profit:' : 'Loss:',
+                hspacer(5),
+                fmtMillions(v, {
+                    prefix: '$',
+                    precision: 2,
+                    label: true,
+                    asElement: true
+                })
+            ];
+        },
 
-    @bindable displayedOrderSymbol = '';
+        labelField: 'name',
+        valueField: 'pnl',
+        heatField: 'pnl',
+        valueFieldLabel: 'Pnl',
```

Comment:
"P&L" please (my OCD always gets tripped up by this one).

We might want to change the default tooltip value format (within the map code) to use formatLedger as well - I think that will be what we want most of the time.